### PR TITLE
add static page content to opt out form

### PR DESCRIPTION
**Pull request by rianfowler**
[Ticket](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/11404)
[Heroku](https://vetsgov-pr-7947.herokuapp.com/education/opt-out-information-sharing/)

Ignore the "Ready to Apply?" button- will address that as part of the opt-out modal. It's using a placeholder widget for now. 

- Left the side nav links to make it consistent with the rest of the `/education` pages. Can remove if necessary
- Used to breadcrumb title as the side nav link title 
- This template inserts a previous link at the bottom of the page that was not in the mock up. LMK if this needs to be removed. 

Todo:

- [x] Add item scope tag back
- [x]  Fix title 
- [x] hide widget 
- [x] remove left nav 
- [x] update url 

download-3

### CODE REVIEW

File: content/pages/education/opt-out.md [hunk 1]
```diff
@@ -1,98 +1,55 @@
...
-      description: If you have a service-connected disability that limits your ability to work or prevents you from working, find out if you can get VR&E benefits and services—like help exploring employment options and getting more training if required.
+order: 10
 widgets:
   - root: react-applicationStatus
     timeout: 20
     loadingMessage: Checking your application status.
     errorMessage: <strong>We’re sorry. Something went wrong when we tried to load your saved application.</strong><br/>Please try refreshing your browser in a few minutes.
 description: Find the right VA education benefits for you, and apply to start getting help paying tuition. We can also help you find the right school or training program.
 ---
-<div itemscope itemtype ="http://schema.org/HowTo">
```
Comment (rianfowler) [inline content/pages/education/opt-out.md]: I dunno what this div is for but I removed it.

**Review by cehsu — APPROVED** [review]
LGTM once the schema tags are replaced (as we probably want these for search engine result pages) and @peggygannon and @emilywaggoner approve.

**Review by peggygannon — CHANGES_REQUESTED** [review]
@rianfowler Please see below for my edits: 
1. the title of the page in the heroku link is: 
Opt Out of Sharing VA Education Benefits Information
Was there a decision to shorten the header? 

2. Please remove the Ready to apply? question above the green button.

3. Green button languages needs to say Opt Out

4. Please remove "the" before VA in this header: 
What kind of information does VA share with schools?

5. Per feedback from Mikki, this page should not appear in Education left nav. We are still discussing entry points for this page. 

6. the URL for this is changing to: vets.gov/education/opt-out-information-sharing

File: content/pages/education/opt-out.md (hunk 1 above)
Comment (rianfowler) [reply content/pages/education/opt-out.md]: Cool I'll add this back.

**Comment by rianfowler** [comment]
@peggygannon Updated the screenshot with the changes.

**Comment by peggygannon** [comment]
@rianfowler Thank you!

It looks like we lost the green Opt Out button. It should go below: 
**How do I tell VA I don’t want my information shared?**
You’ll need to fill out a short form to tell us you want to opt out of sharing this information. You can get started right now.

also, I realize in my earlier comment that I didn't give you the URL to the form. I'm not sure if you have it, but here are the URLs for both the landing page and the form (green button)
The URL for the landing page is: vets.gov/education/opt-out-information-sharing
the URL for the form (green button) is: vets.gov/education/opt-out-information-sharing/opt-out-form-0993

**Comment by rianfowler** [comment]
@peggygannon sorry I uploaded the wrong screen shot.

**Comment by rianfowler** [comment]
@peggygannon Currently I have it set up to show the modal when the opt out button is clicked and the form link on "Yes, I Want to Opt Out" mentioned on this ticket: https://app.zenhub.com/workspace/o/department-of-veterans-affairs/vets.gov-team/issues/11291

@cehsu we'll need to update the app url to this per the comment above: vets.gov/education/opt-out-information-sharing/opt-out-form-0993

**Review by emilywaggoner — CHANGES_REQUESTED** [review]
Thanks @rianfowler! I have one request:

Would you mind swapping the positions of the first and last questions?

Specifically, move the following, including the green button, to the last position:

### How do I tell VA I don't want my information shared?
You'll need to fill out a short form...
[ opt out call to action ]

And then move this to the first position:

### What's the Colmery Act?
The Harry W. Colmery Veterans Educational... 
[Learn more ...]

Edited to loop in @peggygannon and provide more context. I think we should move the CTA to the bottom so that we don't encourage someone to begin the opt out process before they've read through the page content.

**Comment by emilywaggoner** [comment]
@peggygannon That sounds great to me. @rianfowler let's do what Peggy described.

**Comment by rianfowler** [comment]
@peggygannon @emilywaggoner There a little repetition with the header content under the title. 

@peggygannon Regarding this: 

>  but typically for forms, we don't create a FE file for static content pages. 

When we set up this page we assumed that it would need to be part of the education collection and have the side nav- was easier for us to treat it as a static content page. Now that we know we're not doing this, we could move this into an introduction page like we do with our other forms. @cehsu let's discuss this in stand up- I know we made some changes to remove the introduction page and to treat the opt out as a single page form. 

download-4

**Comment by cehsu** [comment]
@peggygannon indicated that we may want to add an amended note under the button recommending that the user log in to be able to save their form, and verify their account to enable prefill (similar to how the form start buttons on the form introduction pages are framed -- using the SIPIntro component). Is there another ticket this should be captured on for discussion @peggygannon?

**Pull request closed without merging**